### DISCUSSION TITLE
Is it possible to use testcontainers (uber) for AsyncAPI mocks and tests?

### DISCUSSION THREAD

**Discussion by akadnikov** [microcks/microcks#1118] [Q&A] [answered]
Hello!

Is it possible to use testcontainers (uber) for AsyncAPI mocks and tests? Are there any examples?

Thanks!

**Comment by lbroudoux** [comment, marked as answer]
Yes, it's possible! And you've got a full demonstration and tutorial here: https://github.com/microcks/microcks-testcontainers-java-spring-demo
😉

**Reply by akadnikov** [reply]
Thanks a lot!
